spras/domino.py

```diff
@@ -151,10 +151,10 @@ def run(network=None, active_genes=None, output_file=None, slice_threshold=None,
         out_modules_dir = Path(out_dir, 'active_genes')
 
         # Concatenate each produced module HTML file into one file
-        with open(output_file, 'w') as fo:
+        with open(output_file, 'w') as f:
```

Comment:
Is this the only variable name `typos` recommended changing? I generally would want us to be able to use any variable names we want.